worlds/dark_souls_3/docs/setup_en.md

```diff
@@ -12,24 +12,25 @@ prompt where you can read information about your run and write any command to in
 
 The randomization is performed by the AP.json file, an output file generated by the Archipelago server.
 
-## Installation Procedures
+## Installation Procedures  
```

Comment:
```suggestion
## Installation Procedures
```
The added `  ` should not do anything here? In that case it's a white-space error.